usage/swo.md

```diff
@@ -1,84 +1,116 @@
 # Serial Wire Output
 
-Serial Wire Output (SWO) allows the target to write tracing and logging to the host without using usb or serial port. All data goes over a single pin, the JTAG TDO pin.
+Serial Wire Output (SWO) allows the target to produce tracing and logging information from your firmware
+without using precious peripherals such as USB or UARTs. Instead, the data is emitted via a single pin,
+which is usually what would be JTAG's TDO, but re-purposed for the purpose.
 
-To set up SWO, you need to
+To set up SWO, you need to:
 
-* connect target and bmp
-* configure the target processor
-* configure bmp
-* set up the viewer
+* Write the target to your probe running BMD
```

Comment:
You mean Wire?

Reply:
we do indeed! we'll go correct that now.